devel/ansible/roles/bodhi/files/bodhi.toml

```diff
@@ -0,0 +1,72 @@
+# A sample configuration for fedora-messaging. This file is in the TOML format.
+# For complete details on all configuration options, see the documentation.
+
+amqp_url = "amqp://"
+
+publish_exchange = "amq.topic"
+
+callback = "bodhi.server.consumers:messaging_callback"
+
+# The topic_prefix configuration value will add a prefix to the topics of every sent message.
+# This is used for migrating from fedmsg, and should not be used afterwards.
+topic_prefix = ""
```

Comment:
You should set `org.fedoraproject.dev` here if you want to subscribe to the messages with a prefix (see the `[[bindings]]` section below.

Reply:
Done, do I need to change anything from the bindings (remove the org.fedoraproject.*.) ?

Reply:
No, they need to match, so you either add the prefix in `topic_prefix` or remove it from the bindings. It should work as it is now.